core-bundle/src/Twig/Runtime/FigureRendererRuntime.php

```diff
@@ -23,7 +23,10 @@
 use Twig\Environment;
 use Twig\Extension\RuntimeExtensionInterface;
 
-final class FigureRendererRuntime implements RuntimeExtensionInterface
+/**
+ * @final
```

Comment:
If we cannot declare the class as final, we should declare it as `@internal`.

Reply:
This would IMHO be a BC break. Final classes aren't meant to be extended (= no BC for internal fields) while internal ones aren't meant to be used (= no BC for the API).

Reply:
Why not keep the `final class` then?

Reply:
:bird: &larr; mocky, the mockingbird

Reply:
Alternatively, we could duplicate the code or wrap it in a service. For this use case the implementation of the `FigureRendererRuntime` already does the right thing, including the automatic wrapping of metadata into an object, so that's why I went for it.

Reply:
changes are reverted, see a07d8b43fdbda28530683d0212b684029713bb2f for refactored solution